.github/workflows/recommend-integration-tests.yml

```diff
@@ -0,0 +1,73 @@
+name: Recommend integration tests
+on:
+  pull_request:
+
+jobs:
+  recommend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+
+      - name: Get source files changes
```

Comment:
https://github.com/dorny/paths-filter can be nice for this too as an alternative 👀

Reply:
Ooh this is going to be very helpful when we need to customise the message by path!